### PR TITLE
added changes to use AndroidX libraries and runtime update 7

### PR DESCRIPTION
**Pull request by puneet-pdx**
added gradle.properties
added changes to use AndroidX libraries and runtime update 7

### CODE REVIEW

**Comment by puneet-pdx** [comment]
@gunt0001 can you please review these changes. The changes are for the app to use runtime Update 7 and Androidx libraries.

File: mapbook-app/src/main/res/menu/menu.xml [hunk 1]
```diff
@@ -12,7 +12,7 @@
     <item android:id="@+id/action_search"
           android:title="@string/search"
           app:showAsAction="ifRoom"
-          app:actionViewClass="android.support.v7.widget.SearchView"
+          app:actionViewClass="androidx.appcompat.widget.SearchView"
           android:icon="@android:drawable/ic_menu_search"/>
 
 </menu>
```
Comment (changanxian) [inline mapbook-app/src/main/res/menu/menu.xml]: adds a blank line

**Comment by changanxian** [comment]
@puneet-pdx looks good to me

File: mapbook-app/src/main/res/menu/menu.xml (hunk 1 above)
Comment (puneet-pdx) [reply mapbook-app/src/main/res/menu/menu.xml]: updated

**Comment by puneet-pdx** [comment]
@eri9600 can you review?

File: gradle.properties [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+# Default value: -Xmx10248m -XX:MaxPermSize=256m
+# org.gradle.jvmargs=-Xmx2048m -XX:MaxPermSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
+
+# When configured, Gradle will run in incubating parallel mode.
+# This option should only be used with decoupled projects. More details, visit
+# http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
+# org.gradle.parallel=true
+
+android.enableJetifier=true
+android.useAndroidX=true
```
Comment (eri9600) [inline gradle.properties]: needs a new line here

**Comment by puneet-pdx** [comment]
@esreli do you wanna look at the new release I added to RELEASE.md file before I go ahead and merge this ?

**Comment by esreli** [comment]
@puneet-pdx sure. LGTM.

**Pull request merged**